bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/StatisticDay.java

```diff
@@ -301,6 +301,7 @@ private ChartToolTipInfo createToolTipData(int valueIndex) {
       final ZonedDateTime zdtTourStart = _tourDayData.allStartDateTimes.get(valueIndex);
       final ZonedDateTime zdtTourEnd = zdtTourStart.plusSeconds(elapsedTime);
 
+      //TODO FB propose pace based on RECORDED TIME ?
```

Comment:
@wolfgang-ch 
What do you think ? should pace be computed only with elapsed time ? Should we give the ability for the user to select which time is used to compute the pace in the preferences ? (Elapsed vs Recorded vs Moving)

Reply:
I'm mainly interrested in speed/pace values which are based on the moving time, the columns in tour book view are based on this value but as a new feature you can provide the user the option to select the base time.

![mt-speed](https://user-images.githubusercontent.com/1283445/93206779-3b2acf00-f75a-11ea-9c92-99845d18ae48.png)

Reply:
Ok, I will give some thought to that.

Reply:
If I add a preference for the user to choose pace from moving time vs pace from recorded time, would that preference menu be a good place for a "Pace" group UI ? do you see a better place ?

![image](https://user-images.githubusercontent.com/25235665/93523992-174dc180-f8f1-11ea-9715-55522ccaa5d0.png)

Reply:
> If I add a preference for the user to choose pace from moving time vs pace from recorded time, would that preference menu be a good place for a "Pace" group UI ? do you see a better place ?

The location where this pref setting should be put on, depends how often it is modified. To put it into a prefpage, the user need more clicks to change it as when it is in a slidout

Reply:
True, I could put it in a slideout but I don't think the user will change that option often so it's probably OK to have it in the preferences.

Reply:
Pref page is OK for me